src/styles/colour/index.md.njk

```diff
@@ -59,7 +59,7 @@ use `govuk-colour("red")` rather than `$govuk-error-colour`.
 
 ## Colour palette
 
-Use these colours for graphs and supporting material.
+Use these colours for supporting material.
```

Comment:
Do we need to give an example of supporting material?

Also, do we need to to explicitly discourage users from using colours in charts and graphs? If so, should we add something like the following?

> We have not provided colours for specific use in charts or graphs. Although you could use them this way, they do not contrast with each other enough for use in a bar chart or similar displays.

Reply:
I think it might help here actually... something like 
'Use these colours for supporting materials like illustrations, or in custom components where appropriate.'

I'm not sure we need to discourage users from using colours in charts and graphs – as with using these colours in any other context, it's how you use them. I think the omission earlier is enough to make sure we're not saying they're designed especially for graphs.

Reply:
```suggestion
Use these colours for supporting materials like illustrations, or in custom components where appropriate.
```